python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -107,7 +111,36 @@ def _on_attribute__set__(self, obj_type, attrib_name):
     def __repr__(self):
         """Get the representation of the file."""
 
-        return "MassStorageFile(namePattern='%s')" % self.namePattern
+        return "%s(namePattern='%s')" % (_getName(self), self.namePattern)
+
+    def mass_line_processor(self, line):
+        lineParts = line.split(' ')
```

Comment:
I think that the default behaviour of split is to split on whitespace so could this simply be

```python
lineParts = line.split()
```